share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

```diff
@@ -1,52 +1,70 @@
 # How To Open a Homebrew Pull Request (and get it merged)
 
-The following commands are used by Homebrew's contributors to setup a fork of Homebrew's Git repository on GitHub, create a new branch and create a GitHub pull request of the changes in that branch.
+The following commands are used by Homebrew contributors to set up a fork of Homebrew's Git repository on GitHub, create a new branch, and create a GitHub pull request ("PR") of the changes in that branch.
 
-Depending on the change you want to make, you need to send the pull request to the corresponding repository. If you want to submit a change in Homebrew core code, you should open the pull request at [Homebrew/brew](https://github.com/Homebrew/brew). If you want to submit a change for a formula, you could open the pull request at [Homebrew core tap](https://github.com/Homebrew/homebrew-core) or any other [official taps](https://github.com/Homebrew) based the formula type.
+Depending on the change you want to make, you need to send the pull request to the appropriate one of Homebrew's main repositories. If you want to submit a change in Homebrew core code (the `brew` implementation), you should open the pull request on [Homebrew/brew](https://github.com/Homebrew/brew). If you want to submit a change for a formula, you should open the pull request on [the `homebrew/core` tap](https://github.com/Homebrew/homebrew-core) or another [official tap](https://github.com/Homebrew), based the formula type.
```

Comment:
Is “you need to send the pull request to the appropriate one of Homebrew's main repositories” supposed to be parsed as “you need to send the pull request to the appropriate repository, which should be one of Homebrew's main/official repositories”? If so, that doesn't really work out for me.

Maybe a simplification like so is sufficient (as the following text goes into detail what is the right repository): “Depending on the change you want to make, you need to send the pull request to the appropriate repository.”